### PR TITLE
Camera perspective memory bar – UI for saving/loading camera positions

### PR DESCRIPTION
**Pull request by benptc**
Trying not to spend too much time for this as it's just a fun convenience feature, but I think I've covered the main functionality:
- Integrates with Dan's existing save/load camera perspectives
- Saves a thumbnail of the current splat canvas or threejs canvas in the selected slot
- Stores the perspectives per-world in localStorage, and loads them back when refreshed
- Includes some error handling to delete old thumbnails from local storage if you've exceeded a certain storage limit
- Can press the `.` key or use the menu bar item to toggle the UI

Requires: https://github.com/ptcrealitylab/vuforia-spatial-toolbox-userinterface/pull/745

camera-position-memory-bar-v5

### CODE REVIEW

File: content_scripts/setupMenuBar.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ import Splatting from '../../src/splatting/Splatting.js';
         DebugAvatarConnections: 'Debug Avatar Connections',
         DeleteAllTools: 'Delete All Tools',
         DownloadScan: 'Download Scan',
+        DownloadZipBackup: 'Download World Object',
```
Comment (hobinjk-ptc) [inline content_scripts/setupMenuBar.js:37]: nit: unrelated change crept in from #201

**Review by hobinjk-ptc — APPROVED** [review]

**Review by Steve-KX-RL — APPROVED** [review]
LGTM!

**Review by dangond-ptc — APPROVED** [review]
LGTM! Very useful feature to have, and great UI for it!

File: content_scripts/setupMenuBar.js (hunk 1 above)
Comment (benptc) [reply content_scripts/setupMenuBar.js:37]: whoops
